attribute-utils/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+    `java-library`
+}
+
+dependencies {
+    implementation(project(":hypertrace-trace-enricher:enriched-span-constants"))
+    implementation("org.hypertrace.core.datamodel:data-model:0.1.9")
+    implementation(project(":span-normalizer:raw-span-constants"))
+    implementation("org.hypertrace.entity.service:entity-service-client:0.1.23")
```

Comment:
Is there any use of client in this lib?

Reply:
This module has got some attributes which will used in next pr.